scripts/package/postinstall.sh

```diff
@@ -68,7 +68,10 @@ The "observiq-otel-collector" service has been configured!
 The collector's config file can be found here: 
   /opt/observiq-otel-collector/config.yaml
 
-For more information on configuring the collector, see the OpenTelemetry docs:
+For more information on configuring the collector, see the docs:
+  https://github.com/observIQ/observiq-otel-collector/blob/main/docs/installation-linux.md#configuring-the-collector
```

Comment:
Hmm I wonder if there is a way to catch whether we update the title instructions at all 🤔 either way should be good at the moment.

Reply:
Yeah, that's a good point. But even if the anchor ends up not existing, it'll still take them to that installation-linux docs, which is probably enough.